2015-Fall/mhartvig/Calculator/Calculator/ViewController.swift

```diff
@@ -79,12 +89,16 @@ class ViewController: UIViewController {
     
     @IBAction func Clear() {
         brain.clear()
+        clearDisplay()
+    }
+    
+    func clearDisplay() {
         displayValue = 0
         isUserTyping = false
         isNumberADecimal = false;
```

Comment:
Maybe `isNumberADecimal` as a computed property, simply checking for the '.' in the display. That way we won't have to remember setting the boolean here and there.